docs/05-infrastructure-nodes.md

```diff
@@ -0,0 +1,318 @@
+# OpenShift Infrastructure Nodes
+The OpenShift subscription model allows customers to run various core
+infrastructure components at no additional charge. In other words, a node
+that is only running core OpenShift infrastructure components is not counted
+in terms of the total number of subscriptions required to cover the
+environment.
+
+OpenShift components that fall into the infrastructure categorization
+include:
+
+* kubernetes and OpenShift control plane services ("masters")
+* router
+* container image registry
+* cluster metrics collection
+* cluster aggregated logging
+* service brokers
+
+Any node running a container/pod/component not described above is considered
+a worker and must be covered by a subscription.
+
+## More MachineSet Details
+In the cluster scaling exercises you explored the use of `MachineSets` and
+adding replicas within them. In the case of an infrastructure node, we want
+to create additional `Machines` that have specific kubernetes labels. Then,
+we can tell the various components to run specifically on hosts with those
+labels.
+
+To accomplish this, you will create additional `MachineSets`. The easiest way
+to do this is to `get` the existing `MachineSets` into a file, and then
+modify them. This is because the `MachineSet` has some details that are
+specific to the AWS region that the cluster is deployed in, like the AWS EC2
+AMI ID. For example, given the following output of `oc get machineset -n
+openshift-cluster-api`:
+
+    NAME                         DESIRED   CURRENT   READY     AVAILABLE   AGE
+    190125-3-worker-us-west-1b   2         2         2         2           3h
+    190125-3-worker-us-west-1c   1         1         1         1           3 
+
+There are two available EC2 AZs into which we can deposit infrastructure
+components (`1b` and `1c`). Take a look at one specifically with `oc get
```

Comment:
You should be able to put `Machine(Set)s` in any zone which has some existing infrastructure set up (a subnet, network routing, ...).  Long-term, it would be nice to have that infrastructure managed.  But even now, I'd guess you have all of that infrastructure set up in us-west-1a for this cluster too, so 1a would be a valid zone choice for your new machines.  On the other hand, following existing machines is always going to be safe, so you can also punt on discussing options for additional zones.

Reply:
In the case of my AWS account I only have access to two AZ in this region. There's an asterisk on the AWS page that talks about west-1 and the 3rd AZ.

I think in the cluster scaling example we talk about how the installer will set up default sets and resources in all AZs. if not we probably should add it.

Reply:
@wking 

further down the page:

```
US West (Northern California) Region
EC2 Availability Zones: 3*
Launched 2009

*New customers can access two EC2 Availability Zones in US West (Northern California).
```